06-building-a-program-start-to-finish/setting-up-linters/output/README.md

```diff
@@ -0,0 +1,11 @@
+#### Output
+- **Using**: Terminal & GitHub
+- **Deliver**: two screenshots of ESLint [Airbnb](https://github.com/airbnb/javascript)  style violations in the terminal and [HoundCI](https://houndci.com/) comment on a PR
+- **With**: *independently*
+
+#### Screenshots of Eslint Airbnb style violations and HoundCI comments
+![ESlint error](https://github.com/andela/test-simulations-mgmt-repo/blob/bayo-kakashi-judensi/06-building-a-program-start-to-finish/setting-up-linters/output/eslint%20error%202.png)
+
+![Eslint Error](https://github.com/andela/test-simulations-mgmt-repo/blob/bayo-kakashi-judensi/06-building-a-program-start-to-finish/setting-up-linters/output/eslint%20error.png)
+
+![Hound CI comment](https://github.com/andela/test-simulations-mgmt-repo/blob/bayo-kakashi-judensi/06-building-a-program-start-to-finish/setting-up-linters/output/hound%20comment.png)
```

Comment:
I see the integration from HoundCI and the other from you IDE, it's not from terminal, kindly address.